scripts/setup.sh

```diff
@@ -13,6 +17,16 @@ if [ $? != "0" ]; then
 fi
 
 printf "\n\rSetting up client side dependancies... \n\r\n\r"
+
+NODE_MAJOR_VERSION=$(node -v | cut -d "." -f 1)
+DESIRED_NODE_VERSION=$(cat "${DIR}/../.nvmrc")
+
+if [[ "${NODE_MAJOR_VERSION}" != *"${DESIRED_NODE_VERSION}"  ]]; then
+  echo -e "${red}Your node version ${NODE_MAJOR_VERSION}" does not match "${DESIRED_NODE_VERSION}"
+  echo -e "${red}Please run 'nvm use' to get the desired node version"
+  exit 1
+fi
+
```

Comment:
This ensures you're running the correct version of node. This is also in the start script